src/itineraryCollapsible/ItineraryCollapsible.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { PureComponent } from 'react'
+import cc from 'classcat'
+import { canUseEventListeners } from 'exenv'
+
+import { color } from '_utils/branding'
+import KEYCODES from '_utils/keycodes'
+import Text, { TextDisplayType, TextTagType } from 'text'
+import Bullet, { BulletTypes } from 'bullet'
+import ItineraryLocation, { computeKeyFromPlace } from 'itineraryLocation'
+
+export interface ItineraryCollapsibleProps {
+  readonly places: Place[]
+  readonly className?: Classcat.Class
+  readonly label?: string
+  readonly ariaLabel?: string
+}
+
+interface ItineraryCollapsibleState {
+  readonly collapsed: boolean
+}
+
+const collapsibleLocationSize = 36 // default size of a collapsible ItineraryLocation
```

Comment:
I'd rather avoid that but I couldn't find a better way to make the transition work from small/hidden size to full size.
See https://css-tricks.com/using-css-transitions-auto-dimensions/ for more explanations